scripts/wheels/windows/build.ps1

```diff
@@ -87,7 +87,7 @@ Write-Host "Starting core build"
 Write-Host "Finished core build"
 
 foreach ($python in $pythons){
-    Write-Host "Begining build for python $($python)"
+    Write-Host "Beginning build for python $($python)"
```

Comment:
Line 83 can be changed to:
Core can be `built` independently of the python version.